src/Uno.UI/UI/Xaml/DependencyPropertyDetails.cs

```diff
@@ -282,6 +282,18 @@ internal void ClearBinding()
 			}
 			else
 			{
+				if (precedence == DependencyPropertyValuePrecedences.Animations && (_flags & Flags.LocalValueNewerThanAnimationsValue) != 0)
```

Comment:
I don't see where you are setting the `Flags.LocalValueNewerThanAnimationsValue` ?

Reply:
nm ... it's not a new flag